docs/data/pages.ts

```diff
@@ -4,22 +4,34 @@ const pages: MuiPage[] = [
   {
     pathname: '/toolpad/getting-started',
     title: 'Getting started',
-    icon: 'CreateIcon',
+    icon: 'DescriptionIcon',
     children: [
-      { pathname: '/toolpad/getting-started/setup' },
-      { pathname: '/toolpad/getting-started/tutorial', title: 'Tutorial' },
+      { pathname: '/toolpad/getting-started/introduction', title: 'Introduction' },
+      { pathname: '/toolpad/getting-started/quickstart/', title: 'Quickstart' },
```

Comment:
```suggestion
      { pathname: '/toolpad/getting-started/quickstart', title: 'Quickstart' },
```

The value of the pathname should not have `/` at the end. I know that this is weird but let's follow the pattern for now (I plan to improve the docs infrastructure in the next quarter 🤞)

The code to make the sidebar active URL is in the `_app.js`

```js
// activePage is passed as React context to the sidebar
const activePage = findActivePage(pages, router.pathname);
```

Reply:
Thanks!